apps/etterlatte-behandling/src/main/resources/db/migration/V63__tilknyttetbehandlingid_generellbehandling.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE generellbehandling ADD COLUMN tilknyttetBehandling UUID;
```

Comment:
tenker denne kan være `tilknyttet_behandling` for å matche `sak_id` i samme tabell

Reply:
ups ja, bra catch

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2277/commits/87e5b05957b34c56b61595cd37cbd8b15947ed2c